### ISSUE TITLE
Broken @keyframes

### ISSUE THREAD

**Issue by subzey** [ChristianMurphy/postcss-combine-duplicated-selectors#68]
The animation `@keyframes` become broken after being processed with postcss-combine-duplicated-selectors 2.2.0

Input file:
```
@keyframes appear {
  0% {
    opacity: 0;
  }
  100% {
    opacity: 1;
  }
}

@keyframes disappear {
  0% {
    opacity: 1;
  }
  100% {
    opacity: 0;
  }
}
```

Output:
```
@keyframes appear {
  0% {
    opacity: 0;
    opacity: 1;
  }
  100% {
    opacity: 1;
    opacity: 0;
  }
}

@keyframes disappear {
}

```

**Comment by ChristianMurphy** [comment]
Thanks for reporting @subzey! :bowing_man: :+1: 
The example input and output is very helpful. :1st_place_medal: 

Would you be interested in investigating and opening a pull request?
If not, I may have some time to research this later this week or sometime next week.

**Comment by ChristianMurphy** [comment]
@subzey Thanks again for reporting this!
This has been patched in version 2.3.0, which has just been released on npm.

**Comment by studentIvan** [comment]
got exactly same problem with the version 10.0.2
